### PR TITLE
Group exceeds limit error handling

### PR DESCRIPTION
**Pull request by il10**
assign tasks to groups error handling

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @il10 has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/activity-feed/task-form/__tests__/__snapshots__/TaskError.test.js.snap [hunk 1]
```diff
@@ -10,36 +10,7 @@ exports[`components/content-sidebar/activity-feed/task-form/TaskError when type
...
-        <FormattedMessage
-          defaultMessage="An error occurred while creating this task. Please try again."
-          id="be.api.taskCreateErrorMessage"
-        >
-          <div />
-        </FormattedMessage>
-      </div>
-    </div>
-  </InlineNotice>
+  <div />
```
Comment (il10) [inline src/elements/content-sidebar/activity-feed/task-form/__tests__/__snapshots__/TaskError.test.js.snap]: There should be one new snapshot, but many of them have been replaced with empty containers.

File: src/api/Feed.js [hunk 2]
```diff
@@ -615,18 +618,60 @@ class Feed extends Base {
...
-            errorCallback: (e: ElementsXhrError, code: string) => {
-                this.feedErrorCallback(false, e, code);
-            },
+        // Pull out groups from assignees
+        const groupAssignees = assignees.filter(
+            (assignee: SelectorItem<UserMini | GroupMini>) => (assignee.item && assignee.item.type) === 'group',
+        );
+
+        // Pull out group id from groupAssignees array
+        const groupIds = groupAssignees.map(assignee => assignee.id);
```
Comment (il10) [inline src/api/Feed.js]: we can combine this with LN635 and chain the .map calls into one statement

File: src/api/__tests__/Groups.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    file,
+                    group,
+                    successCallback,
+                    errorCallback,
+                });
+
+                // getGroupCount() returns Promise
+                // Verify whether groups.get resolves or rejects
+                // cannot use toBeCalledWith()
+                expect(groups.get).toBeCalledWith({
```
Comment (il10) [inline src/api/__tests__/Groups.test.js]: Let's use jest.any here, on the successCallback and errorCallback lines. You'll want to check to see if some function is called.

File: src/api/__tests__/Feed.test.js [hunk 4]
```diff
@@ -569,6 +580,84 @@ describe('api/Feed', () => {
...
+        const taskCompletionRule = 'ALL_ASSIGNEES';
+        const dueAt = null;
+
+        const code = 'group_exceeds_limit';
+        const hasError = false;
+        beforeEach(() => {
+            feed.feedErrorCallback = jest.fn();
+        });
+
+        test('should check group size by calling groups endpoint', async () => {
```
Comment (il10) [inline src/api/__tests__/Feed.test.js]: Feed tests added.

**Comment by booc0mtaco** [comment]
creating a new PR to restructure the content here (squash commits and will repost)

**Comment by booc0mtaco** [comment]
we can close this out now

**Pull request closed without merging**